### ISSUE TITLE
Games Almost NEVER launch!

### ISSUE THREAD

**Issue by Multipixelone** [WilliamLCobb/GC4iOS#26]
I used XCode to build the latest version onto my iPhone SE, and copied roms over with iFunBox, but games just WON'T launch! I only got Wind Waker to launch one time, and it crashed when it opened!

Am I missing something? Thanks for any reply 😄

**Comment by Multipixelone** [comment]
Okay, I got more information. It turns out that games only launch when I use XCode to attach to the process... and then the games won't show a title screen. So, I have no clue at this point. If any of you have any more information, please share!

**Comment by DarknesGaming** [comment]
I am still waiting for @WilliamLCobb to look at my work. If Dolphin 5.0 only works on 64-bit devices, and GC4iOS is a port for that emulator, then GC4iOS should only work on 64-bit devices.

**Comment by zakusa** [comment]
help my games almost never launch

**Comment by zakusa** [comment]
actually it never launched before on my ipad air 2

**Comment by GingerOfOz** [comment]
Jailbroken?  Not?  Some games just won't start for me, and the only ones that are "playable" are the Mario Parties.  Double Dash is the only game I've found that won't start at all, but I'm pretty sure your problem is simply that this emulator is in Alpha, and it may be a while before it gets an update.  Sorry I can't help more.

**Comment by Multipixelone** [comment]
Not jailbroken, but I will have to try Mario Party 5 and get back to you.

**Comment by GingerOfOz** [comment]
You might want to just jailbreak.  Pangu just released one that works on the latest iOS.

**Comment by Multipixelone** [comment]
Would jailbreaking do anything bad to my device? It's my main phone and last time I jailbroke an old ipod it didn't work out too well. :/

**Comment by Multipixelone** [comment]
Mario Party 5 results in the same thing as the rest of the games, just FYI.

**Comment by GingerOfOz** [comment]
There's always some level of risk involved, but I've never met anyone who had a device ruined.  Make sure you don't use pirated sources on Cydia, be careful what you download, and you should be fine.  I have had some scary moments with my iPad though...

**Comment by Multipixelone** [comment]
Do you mind me asking about your iPad?

**Comment by GingerOfOz** [comment]
Had a file transferring app that worked fine, but one time I tried to open it, it got stuck, so I put my iPad to sleep. It wouldn't turn back on for a while, and when it did, my password didn't work, it didn't show the time, and the sides of the screen was pixely and distorted.  I tried a few things, nothing worked, and I was hesitant to do a hard reset because people had reported that it sent their devices into a boot loop. (Later people figured out how to get out of it.). Did the reset and my iPad was fine, but ever since then I've been a bit more cautious about where I get my apps.

**Comment by Multipixelone** [comment]
Yeah my iPod got stuck in a boot loop for a while, but I'll have to think about if I want to go down that path, thanks though!
